how-to/workspace-platform-starter/public/oauth_redirect.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```

Comment:
If we don't include the msft integration by default we might not need the oauth_redirect...although it will be added for microflow support.

Reply:
Removed